src/test/cls_hello/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+add_executable(ceph_test_cls_hello
+  test_cls_hello.cc
+  )
+set_target_properties(ceph_test_cls_hello PROPERTIES COMPILE_FLAGS
+  ${UNITTEST_CXX_FLAGS})
+target_link_libraries(ceph_test_cls_hello
+  librados
+  global
+  ${EXTRALIBS}
+  ${BLKID_LIBRARIES}
+  ${CMAKE_DL_LIBS}
+  radostest
+  ${UNITTEST_LIBS}
+  )
+
```

Comment:
I'd leave anything out of make check if it's not there in automake - originally we tried to keep `make check` only running unit tests, later added more functionality when it started getting run automatically on PRs, and then had to start taking out some less useful tests since it became too long.

So there are plenty of ceph_test_* programs that shouldn't be in `make check`, while all of unittest_* should be included